test/Target/invalid-cpp.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: emitc-translate -split-input-file -mlir-to-cpp -verify-diagnostics %s
+
+// expected-error@+1 {{cannot emit tensor type with non static shape}}
+func @non_static_shape(%arg0 : tensor<?xf32>) {
+  return
+}
```

Comment:
Add a test for unranked tensors `tensor<*xf32>`

Reply:
And so it shall be.